EE.C

```diff
@@ -0,0 +1,478 @@
+   TH1F *dilep_mass_DY = new TH1F("dilep_mass_DY","dilepton mass",100,0,1000);
+   
+   TH1F *SBM_DY = new TH1F("SBM_DY","max(SBM)",100,0,300);
+   TH1F *METvsMmm_DY = new TH1F("METvsMmm_DY","E^{miss}_{T}/M(#mu^{+}#mu^{-})",100,0,1);
+   TH1F *HTLT_DY = new TH1F("HTLT_DY","HT-LT",100,-500,500);
+
+   TH1F *Mass_DY = new TH1F("Mass_DY","Lepton mass(M_{e})",100,0,500);
+   
+   TH2F *njvsnbj_DY = new TH2F("njvsnbj_DY","njet vs nbjet",10,0,10,5,0,5);
+//   TH2F *bjetpT_dR_DY = new TH2F("bjetpT_dR_DY","b-jet pT vs DeltaR",100,0,500,50,0,5);
+//   TH2F *bjetpT_elepT_DY = new TH2F("bjetpT_elepT_DY","b-jet pT vs electron pT",100,0,500,100,0,500);
+//   TH2F *bjetpT_mupT_DY = new TH2F("bjetpT_mupT_DY","b-jet pT vs muon pT",100,0,500,100,0,500);
+   
+   Long64_t nentries = fChain->GetEntriesFast();
+
+   float totalN = 122055388;
+   float Lumi = 36.3;
+   float cs = 5765.4;
+   float weight = Lumi * cs/totalN;
+
+   Long64_t nbytes = 0, nb = 0;
+   for (Long64_t jentry=0; jentry<nentries;jentry++) {
+      Long64_t ientry = LoadTree(jentry);
+      if (ientry < 0) break;
+      nb = fChain->GetEntry(jentry);   nbytes += nb;
+      // if (Cut(ientry) < 0) continue;
+
+      run_DY->Fill(run);
+
+      //MET_DY->Fill(met_pt);
+      //dilep_mass_DY->Fill(dil_mass);
+
+      if(lep_pt[0] > 53 &&lep_pt[1] > 53){
+      if(dil_dR > 0.1){
+      if(fabs(lep_dz[0])<0.5 && fabs(lep_dz[1])<0.5){
+
+	  
+	 
+	  //Opposite sign
+	  if(lep_id[0] * lep_id[1] < 0){
+	 
+      //if(dil_chosen != 0) continue;
+	      //particle id
+	      int id1 = lep_id[0];
+	      int id2 = lep_id[1];
+
+	      id_DY->Fill(id1);
+	      id_DY->Fill(id2);
+	      
+	      float Lep_pt[2] = {-999,-999};
+	      float Lep_eta[2] = {-999,-999};
+	      float Lep_phi[2] = {-999,-999};
+	      float Lep_Iso[2] = {-999,-999};
+	      float Lep_E[2] = {-999,-999};
+
+	      float Dil_dPhi[1] = {-999};
+	      float Dil_dR[1] = {-999};
+	      float Dil_mass[1] = {-999};
+	      
+	      float MET[1] = {-999};
+	      
+	      int Njet = 0;
+	      int Nbjet = 0;
+	      int Nnon_bjet = 0;
+	      
+	      vector<float> bJet_pt; bJet_pt.clear();
+	      vector<float> bJet_eta; bJet_eta.clear();
+	      vector<float> bJet_phi; bJet_phi.clear();
+	      vector<float> bJet_E; bJet_E.clear();
+
+	      vector<float> Jet_pt; Jet_pt.clear();
+	      vector<float> Jet_eta; Jet_eta.clear();
+	      vector<float> Jet_phi; Jet_phi.clear();
+	      vector<float> Jet_E; Jet_E.clear();
+	//if(nJets==0)cout<<"Jet: "<<nJets<<endl;
+		  for(int i=0; i<nJets; i++){
+		      if(jet_pt[i] > 30){ //jet pT > 30GeV
+			  if(jet_eta[i] < 5.0){ // jet |eta| <5.0
+
+			      Lep_pt[0] = lep_pt[0];
+
+			      //Number of jet
+			      Njet++;
+
+			      if(jet_btag[i] >= 0.8484) { //b-tagging(WP=0.8484)
+				  Nbjet++;
+				  bJet_pt.push_back(jet_pt[i]);
+				  bJet_eta.push_back(jet_eta[i]);
+				  bJet_phi.push_back(jet_phi[i]);
+				  bJet_E.push_back(jet_E[i]);
+			      }
+			      else Nnon_bjet ++;
```

Comment:
This should be generated out of Njet-Nbjet, there's no need for a third variable.